src/jvm/test/resources/expected_logs/run_concurrent_test/anonymous_object_jdk8.txt

```diff
@@ -0,0 +1,28 @@
+	at java.lang.Thread.run(Thread.java:750)
+
+The following interleaving leads to the error:
+| -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|                                                                                                    Thread 1                                                                                                    |
+| -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| AnonymousObjectRunConcurrentRepresentationTest#1.block(): threw IllegalStateException at BaseRunConcurrentRepresentationTest$testRunWithModelChecker$result$1$1.invoke(RunConcurrentRepresentationTests.kt:40) |
+| -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+
+Detailed trace:
+| -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|                                                                                                    Thread 1                                                                                                    |
+| -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| AnonymousObjectRunConcurrentRepresentationTest#1.block(): threw IllegalStateException at BaseRunConcurrentRepresentationTest$testRunWithModelChecker$result$1$1.invoke(RunConcurrentRepresentationTests.kt:40) |
+|   block(): threw IllegalStateException at AnonymousObjectRunConcurrentRepresentationTest.block(RunConcurrentRepresentationTests.kt:183)                                                                        |
+|     AnonymousObjectRunConcurrentRepresentationTest.runnable = <unknown>#1 at AnonymousObjectRunConcurrentRepresentationTest.block(RunConcurrentRepresentationTests.kt:204)                                     |
+|     AnonymousObjectRunConcurrentRepresentationTest.runnable ➜ <unknown>#1 at AnonymousObjectRunConcurrentRepresentationTest.block(RunConcurrentRepresentationTests.kt:205)                                     |
+|     <unknown>#1.run() at AnonymousObjectRunConcurrentRepresentationTest.block(RunConcurrentRepresentationTests.kt:205)                                                                                         |
```

Comment:
If we have two anonymous class instances for different classes, then both representations are likely `<unknown>#1`, with `#1` in both cases. Is it better to fix the counter or simply not show the caller name?

Reply:
Or let's do nothing :)

Reply:
Agreed to remove the counter in this case.

Reply:
Done